cves/CVE-2012-2774.yml

```diff
@@ -171,10 +178,14 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerability was discovered by users testing the new multithreading
```

Comment:
Where did you see that it was noticed by users testing? You can add a link to where you saw it.

Reply:
"Testing" is a strong word. A user was messing around with multithreading settings and noticed the inconsistency. I'll add a link to the bug report in this answer.